src/solver/expressions/visitors/AstDOTStyleVisitor.cpp

```diff
@@ -0,0 +1,204 @@
+/*
+** Copyright 2007-2024, RTE (https://www.rte-france.com)
+** See AUTHORS.txt
+** SPDX-License-Identifier: MPL-2.0
+** This file is part of Antares-Simulator,
+** Adequacy and Performance assessment for interconnected energy networks.
+**
+** Antares_Simulator is free software: you can redistribute it and/or modify
+** it under the terms of the Mozilla Public Licence 2.0 as published by
+** the Mozilla Foundation, either version 2 of the License, or
+** (at your option) any later version.
+**
+** Antares_Simulator is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+** Mozilla Public Licence 2.0 for more details.
+**
+** You should have received a copy of the Mozilla Public Licence 2.0
+** along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
+*/
+
+#include "antares/solver/expressions/visitors/AstDOTStyleVisitor.h"
+
+#include <antares/solver/expressions/nodes/ExpressionsNodes.h>
+
+namespace Antares::Solver::Visitors
+{
+
+static constexpr BoxStyle SumStyle{"aqua", "hexagon", "filled, solid"};
+static constexpr BoxStyle BinaryStyle{"aqua", "oval", "filled, rounded"};
+static constexpr BoxStyle ComparisonStyle{"yellow", "diamond", "filled"};
+static constexpr BoxStyle NegationStyle{"tomato", "invtriangle", "filled, solid"};
+static constexpr BoxStyle LiteralStyle{"lightgray", "box", "filled, solid"};
+static constexpr BoxStyle VariableStyle{"gold", "box", "filled, solid"};
+static constexpr BoxStyle ParameterStyle{"wheat", "box", "filled, solid"};
+static constexpr BoxStyle ComponentParameterStyle{"springgreen", "octagon", "filled, solid"};
+static constexpr BoxStyle ComponentVariableStyle{"goldenrod", "octagon", "filled, solid"};
+static constexpr BoxStyle PortFieldStyle{"olive", "component", "filled, solid"};
```

Comment:
Use the anonymous namespace
```suggestion
namespace {
constexpr BoxStyle SumStyle{"aqua", "hexagon", "filled, solid"};
constexpr BoxStyle BinaryStyle{"aqua", "oval", "filled, rounded"};
constexpr BoxStyle ComparisonStyle{"yellow", "diamond", "filled"};
constexpr BoxStyle NegationStyle{"tomato", "invtriangle", "filled, solid"};
constexpr BoxStyle LiteralStyle{"lightgray", "box", "filled, solid"};
constexpr BoxStyle VariableStyle{"gold", "box", "filled, solid"};
constexpr BoxStyle ParameterStyle{"wheat", "box", "filled, solid"};
constexpr BoxStyle ComponentParameterStyle{"springgreen", "octagon", "filled, solid"};
constexpr BoxStyle ComponentVariableStyle{"goldenrod", "octagon", "filled, solid"};
constexpr BoxStyle PortFieldStyle{"olive", "component", "filled, solid"};
}
```

Reply:
That’s code smell